libraries/botbuilder-ai/src/qnamaker-utils/httpRequestUtils.ts

```diff
@@ -45,14 +45,20 @@ export class HttpRequestUtils {
 
         const headers: any = this.getHeaders(endpoint);
 
-        const qnaResult: any = await fetch(requestUrl, {
-            method: 'POST',
-            headers: headers,
-            timeout: timeout,
-            body: payloadBody
-        });
-        
-        return await qnaResult.json();
+        try {
+            const qnaResult = await fetch(requestUrl, {
+                method: 'POST',
+                headers: headers,
+                timeout: timeout,
+                body: payloadBody
+            });
+            const jsonResult = qnaResult.status == 204 ? this.getSuccessfulTrainApiResult() : await qnaResult.json();
+
+            return jsonResult;
+
+        } catch (error) {
+            throw error;
+        }
```

Comment:
The catch only rethrows the error, so it is unnecessary.

Reply:
removed trycatch